### PR TITLE
feat: download .json with data from app

### PR DESCRIPTION
**Pull request by Kovalenko77**

### CODE REVIEW

File: src/DownloadFile.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Item } from './types';
+
+interface DownloadFileProps {
+  items: Item[];
+}
+
+const DownloadFile = ({ items }: DownloadFileProps) => {
+  function download(content: string, fileName: string, contentType: string) {
```
Comment (romanalesenkov1) [inline src/DownloadFile.tsx]: why do you use different type of functions here and below?

File: src/index.css [hunk 2]
```diff
@@ -87,3 +85,7 @@ td {
 .tracker-table-foot {
   background-color: #d4dff9;
 }
+
+.download-button {
+  background-color: rgb(20, 130, 73);
```
Comment (romanalesenkov1) [inline src/index.css]: why do you use different type of colors?
sometimes it is #d4dff9, and sometimes it is rgb(20, 130, 73) ?

Comment (Kovalenko77) [reply src/index.css]: Done. Thank you.

File: src/DownloadFile.tsx (hunk 1 above)
Comment (Kovalenko77) [reply src/DownloadFile.tsx]: Done. Thank you.

**Pull request merged**